examples/ensemble/plot_stack_predictors.py

```diff
@@ -0,0 +1,102 @@
+"""
+=================================
+Combine predictors using stacking
+=================================
+
+Stacking refers to a method to blend estimators. In this strategy, some
+estimators are individually fitted on some training data while a final
+estimator is trained using the stacked predictions of these base estimators.
+
+In this example, we illustrate the use case in which different regressors
+(linear and non-linear) are stacked together and a final non-linear regressor
+is used to output the prediction. We compare the performance of each individual
+regressor with the stacking strategy. Stacking improves the overall
+performance.
```

Comment:
stacking does not improve the performance according to https://70811-843222-gh.circle-artifacts.com/0/doc/auto_examples/ensemble/plot_stack_predictors.html
Lasso R^2 0.34, Ridge R^2 0.36, Stacking R^2 0.16